# Author: Caspar van Leeuwen
# Institute: SURF
# Increase timeout for c10d gloo process group operations since test_allreduce_coalesced_basics
# was failing with a timeout (see https://github.com/easybuilders/easybuild-easyconfigs/pull/15137)
diff -Nru pytorch/test/distributed/test_c10d_gloo.py pytorch_orig/test/distributed/test_c10d_gloo.py
--- pytorch/test/distributed/test_c10d_gloo.py	2022-04-19 15:27:48.540163735 +0200
+++ pytorch_orig/test/distributed/test_c10d_gloo.py	2022-04-07 18:31:13.110755000 +0200
@@ -216,7 +216,7 @@
 
     def opts(self, threads=2):
         opts = c10d.ProcessGroupGloo._Options()
-        opts._timeout = 5.0
+        opts._timeout = 50.0
         opts._devices = [create_device(interface=LOOPBACK)]
         opts._threads = threads
         return opts
